server/api/config.go

```diff
@@ -109,7 +109,11 @@ func (h *confHandler) Post(w http.ResponseWriter, r *http.Request) {
 
 	// if ttlSecond defined, we will apply if to temp configuration.
 	if ttls > 0 {
-		h.svr.SaveTTLConfig(conf, time.Duration(ttls)*time.Second)
+		err := h.svr.SaveTTLConfig(conf, time.Duration(ttls)*time.Second)
```

Comment:
seem we can set the TTL for all persist configs but the TTL can only work for some of them

Reply:
Maybe we should return http.StatusBadRequest if any unsupported config is posted?

Reply:
> There is still a comment which is not resolved
I thought maybe it should in a seperated PR. And currently, the temporary settings won't be visible in our http api, a PR should also be sent to solve this problem.
I'll open an issue after this one get merged.